src/controllers/expense.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const expenseService = require('../services/expense');
+const userServise = require('../services/user');
+
+const getFiltered = (req, res) => {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = req.query;
+
+  const expenses = expenseService.getFiltered(+userId, categories, from, to);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { expenseId } = req.params;
+
+  if (!expenseId) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundExpense = expenseService.getById(+expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const addExpense = (req, res) => {
```

Comment:
Unnecessary to add some names to the `add` or `delete`
```suggestion
const add = (req, res) => {
```